react-native/training/src/app/screens/TodoList/components/Title/index.js

```diff
@@ -1,20 +1,18 @@
-import React, { Component } from 'react';
+import React from 'react';
```

Comment:
This component is repeated. I'd delete one of these, and extract to `app/components`